.changeset/fine-apes-switch.md

```diff
@@ -0,0 +1,7 @@
+---
+'@xstate/store': patch
+---
+
+Fix the types for `useAtom` to accept `ReadonlyAtom` values.
+
+See #5306
```

Comment:
nit: feels redundant here - the given changeset entry is pretty much self-explanatory
```suggestion
```